azure-pipelines.yml

```diff
@@ -52,7 +52,7 @@ stages:
     - script: make  EXTRA_FLAGS=-Werror test-all
       displayName: 'Run all unit tests'
     - script: make  EXTRA_FLAGS=-Werror all
-      displayName: 'Build all oficial targets'
+      displayName: 'Build all official targets'
     - script: mkdir release; cp obj/*.hex release/
       displayName: 'Copy artefacts'
```

Comment:
Not sure about this one. Azure uses "artifacts" but "artefacts" is also correct. It's the Queen's english.

Reply:
To me is a "technical" word, so I prefer to use the same than azure, but as you say, both are valid.

Reply:
All right. "Fixed" that one too. 🙂